src/containers/ClusterTriggerBindings/ClusterTriggerBindings.js

```diff
@@ -55,6 +49,15 @@ export /* istanbul ignore next */ class ClusterTriggerBindings extends Component
     }
   }
 
+  getError() {
+    const { error } = this.props;
+    if (error) {
+      return { error };
```

Comment:
Looks like my nit didn't save - is error always just a message here ? if so should we call it message given the object gets passed around as error

Reply:
Today it is, but that won't be the case for long. With https://github.com/tektoncd/dashboard/issues/1842 we'll have `Error` objects or similar with more information. Trying to get things cleaned up before that so it's a bit easier to manage the switch.